sale_order_packaging_info/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field
+            name="name"
+        >res.config.settings.view.form.inherit.sale_order_packaging_info</field>
+        <field name="model">res.config.settings</field>
+        <field name="priority" eval="10" />
+        <field name="inherit_id" ref="sale.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//div[@id='sale_config_online_confirmation_sign']/.."
+                position="inside"
+            >
+                <div
+                    class="col-12 col-lg-6 o_setting_box"
+                    id="sale_order_packaging_info_products"
+                >
+                    <div class="o_setting_right_pane">
+                        <label for="packaging_product_ids" />
+                        <div class="text-muted">
+                            Products that are added to the packaging section of a sales order by default.
+                        </div>
+                        <div class="text-muted">
```

Comment:
i think that the class should be `content-group` instead of `text-muted`, which is for descriptions.

Reply:
I had copied this from the `sale` module:

```xml
<div class="col-12 col-lg-6 o_setting_box">
                            <div class="o_setting_left_pane"/>
                            <div class="o_setting_right_pane">
                                <span class="o_form_label">Down Payments</span>
                                <div class="text-muted">
                                    Product used for down payments
                                </div>
                                <div class="text-muted">
                                    <field name="deposit_default_product_id"/>
                                </div>
                            </div>
                        </div>
```

Broadly you appear to be right, though. I hate modifying res_config_settings views...